src/application/scene/fxml/Drawboard.fxml

```diff
@@ -1,10 +1,123 @@
+               <children>
+                  <HBox prefHeight="60.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="350.0">
+                     <children>
+                        <Button fx:id="buttonDashboard" graphicTextGap="20.0" minHeight="-Infinity" minWidth="-Infinity" mnemonicParsing="false" onAction="#dashboard" prefHeight="60.0" prefWidth="200.0" styleClass="button_tool" stylesheets="@../css/Drawboard.css" text="dashboard">
+                           <graphic>
+                              <ImageView fitHeight="30.0" fitWidth="30.0" pickOnBounds="true" preserveRatio="true">
+                                 <image>
+                                    <Image url="@../assets/icon_dashboard_white.png" />
+                                 </image>
+                              </ImageView>
+                           </graphic>
+                        </Button>
+                        <Region minWidth="0.0" prefHeight="60.0" prefWidth="197.0" />
+                        <TextField fx:id="textFieldTitle" alignment="CENTER" prefHeight="40.0" prefWidth="500.0" styleClass="text_field_title" stylesheets="@../css/Drawboard.css" text="MY DRAWING (CLICK TO CHANGE TITLE)">
+                           <font>
+                              <Font size="18.0" />
+                           </font>
+                           <HBox.margin>
+                              <Insets top="10.0" />
+                           </HBox.margin>
+                        </TextField>
+                     </children>
+                  </HBox>
+                  <HBox alignment="CENTER" prefHeight="60.0" prefWidth="280.0" AnchorPane.rightAnchor="0.0">
+                     <children>
+                        <ToggleButton fx:id="buttonBrush" prefHeight="60.0" prefWidth="60.0" selected="true" styleClass="button_tool" stylesheets="@../css/Drawboard.css">
+                           <graphic>
+                              <ImageView fitHeight="30.0" fitWidth="30.0" pickOnBounds="true" preserveRatio="true">
+                                 <image>
+                                    <Image url="@../assets/icon_drawboard_paintbrush.png" />
+                                 </image>
+                              </ImageView>
+                           </graphic>
+                           <HBox.margin>
+                              <Insets right="10.0" />
+                           </HBox.margin>
+                        </ToggleButton>
+                        <ToggleButton fx:id="buttonEraser" prefHeight="60.0" prefWidth="60.0" styleClass="button_tool">
+                           <graphic>
+                              <ImageView fitHeight="30.0" fitWidth="30.0" pickOnBounds="true" preserveRatio="true">
+                                 <image>
+                                    <Image url="@../assets/icon_drawboard_eraser.png" />
```

Comment:
this image doesn't appear to be in the files, so its not showing up on the screen. Can it be added in pls?

Reply:
fixed, added image to assets